concourse/pipelines/create-deployer.yml

```diff
@@ -467,6 +467,10 @@ jobs:
                               ./paas-cf/manifests/concourse-manifest/extensions/datadog-agent.yml
                               ./paas-cf/manifests/shared/deployments/datadog-agent.yml"
             fi
+            # shellcheck disable=SC2050
```

Comment:
I disagree it is not more changes, it is the same amount, and it would be consistent with the other ways to define environment specific configuration.